whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -195,6 +195,11 @@ public static Message cantChangeOrgName() {
                 "Please contact \"ncc@ripe.net\" to change the name.");
     }
 
+    public static Message cantChangeCountryCode() {
```

Comment:
Why not re-use the existing error message:
`Attribute \"country:\" can only be changed by the RIPE NCC for this object. Please contact \"ncc@ripe.net\" to change it."`